client/packages/programs/src/VaccineCourseEditModal/VaccineCourseEditModal.tsx

```diff
@@ -259,6 +259,7 @@ const VaccineCourseDoseTable = ({
           // temp - will be overwritten by the backend to assign unique dose number (even if previous doses were deleted)
```

Comment:
Side note but I think we should remove this comment, we don't have a dose number at all anymore!